### DISCUSSION TITLE
ERROR in tree with C++

### DISCUSSION THREAD

**Discussion by Fredigarthegreat** [nvim-treesitter/nvim-treesitter#5661] [Q&A]
Not sure why these errors are occurring. They are really messing up my colors. I already did :TSUpdate. Didn't change anything. Any ideas?
![nvim treesitter error](https://github.com/nvim-treesitter/nvim-treesitter/assets/11061437/19475598-1f13-46c8-80dd-f987f87665e0)

**Comment by amaanq** [comment]
not an issue here, just that tree-sitter-c won't parse custom macros all that well.

**Reply by Fredigarthegreat** [reply]
Thanks for helping :)

Is that really what's going on? It's coloring a string as a type.

**Reply by Fredigarthegreat** [reply]
Back to Vscode I guess 
![download](https://github.com/nvim-treesitter/nvim-treesitter/assets/11061437/99497ebb-2a16-4654-a827-06aaf751daf3)

**Reply by amaanq** [reply]
If a string is colored as a type that's probably an issue w/ your colorscheme, I highly doubt error recovery went that badly w/ tree-sitter
